tests/brokers/base/connection.py

```diff
@@ -46,3 +46,21 @@ async def test_connect_by_url_priority(self, settings):
         await broker.connect(**kwargs)
         assert await self.ping(broker)
         await broker.close()
+
+    @pytest.mark.asyncio()
+    async def test_connection_alive(self, settings):
```

Comment:
I don't think, that we should create an extra tests. We already have `self.ping` method in the suite. Just use `broker.ping` inside.
But, we should add a test to check ping timeout